app/src/features/surveys/observations/SurveyObservationPage.tsx

```diff
@@ -0,0 +1,59 @@
+import Box from '@mui/material/Box';
+import CircularProgress from '@mui/material/CircularProgress';
+import { grey } from '@mui/material/colors';
+import { SurveyContext } from 'contexts/surveyContext';
+import { useContext } from 'react';
+import { ObservationComponent } from './ObservationComponent';
+import { SamplingSiteList } from './sampling-sites/SamplingSiteList';
+import { SurveyObservationHeader } from './SurveyObservationHeader';
+
+export const SurveyObservationPage = () => {
+  const surveyContext = useContext(SurveyContext);
+
+  if (!surveyContext.surveyDataLoader.data) {
+    return <CircularProgress className="pageProgress" size={40} />;
+  }
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      height="100%"
+      sx={{
+        background: '#fff'
+      }}>
+      <Box
+        zIndex={999}
```

Comment:
This seems kind of hacky, is it needed? Can we use `zIndex={2}` or something realistic?

Reply:
the map component sits at a zindex of ~800 so we needed something drastic to keep the sticky header above the map on the add sampling site page